web/src/js/components/contentviews/HttpMessage.tsx

```diff
@@ -141,15 +141,18 @@ type ViewImageProps = {
     flow: HTTPFlow;
     message: HTTPMessage;
 };
-
+//added svg and other format image previes support
 export function ViewImage({ flow, message }: ViewImageProps) {
+    const imageContentURL: string = MessageUtils.getContentURL(flow, message);
+
     return (
         <div className="flowview-image">
-            <img
-                src={MessageUtils.getContentURL(flow, message)}
-                alt="preview"
-                className="img-thumbnail"
-            />
+            <object
+                data={imageContentURL}
+                type="image/*"
+                width="100%"
+                height="100%"
+            ></object>
```

Comment:
I don't understand this here - why does the img tag not work?

Reply:
@mhils  kindly  review PR #6551  ,